tests/defer/tests.py

```diff
@@ -271,3 +272,15 @@ def test_custom_refresh_on_deferred_loading(self):
             # access of any of them.
             self.assertEqual(rf2.name, 'new foo')
             self.assertEqual(rf2.value, 'new bar')
+
+
+class TestDefer3(AssertionMixin, TestCase):
+    def test_field_not_overridden_by_parent(self):
+        FirstChildModel.objects.create()
```

Comment:
I think a docstring would be useful to explain the expected behavior. See [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style) for docstring guidelines.